onnxruntime/python/tools/quantization/quantize.py

```diff
@@ -370,6 +370,12 @@ def quantize_static(
                     `com.microsoft` domain, which forces use of ONNX Runtime's QuantizeLinear and DequantizeLinear
                     contrib op implementations. The contrib op implementations may support features not standardized
                     into the ONNX specification (e.g., 16-bit quantization types).
+                QuantMinRealRange = float|None :
```

Comment:
Any suggestions for a more descriptive name?

Reply:
How about MinimumRealRange?

Reply:
I made the change, thanks.